### PR TITLE
Fixing compatibility with Create mod version 0.5.1

### PR DESCRIPTION
**Pull request by dima-dencep**
In 0.5.1 they changed the package of some classes, without this fix in the console output:

```sh
[14:38:06] [modloading-worker-0/ERROR]: Failed to load: mod.chiselsandbits.forge.compat.create.CreateCandBPlugin
java.lang.NoClassDefFoundError: com/simibubi/create/content/contraptions/components/structureMovement/MovementBehaviour
	at java.lang.Class.forName0(Native Method) ~[?:?]
	at java.lang.Class.forName(Class.java:375) ~[?:?]
	at mod.chiselsandbits.api.util.ClassUtils.createOrGetInstance(ClassUtils.java:45) ~[chisels-and-bits-forge-1.3.135.jar%23300!/:1.3.135]
	at mod.chiselsandbits.forge.platform.ForgePluginDiscoverer.createPluginFrom(ForgePluginDiscoverer.java:95) ~[chisels-and-bits-forge-1.3.135.jar%23300!/:1.3.135]
	at mod.chiselsandbits.forge.platform.ForgePluginDiscoverer.loadPlugins(ForgePluginDiscoverer.java:58) ~[chisels-and-bits-forge-1.3.135.jar%23300!/:1.3.135]
	at mod.chiselsandbits.plugin.PluginManger.detect(PluginManger.java:35) ~[chisels-and-bits-forge-1.3.135.jar%23300!/:1.3.135]
	at mod.chiselsandbits.ChiselsAndBits.<init>(ChiselsAndBits.java:62) ~[chisels-and-bits-forge-1.3.135.jar%23300!/:1.3.135]
	at mod.chiselsandbits.forge.Forge.lambda$new$1(Forge.java:42) ~[chisels-and-bits-forge-1.3.135.jar%23300!/:1.3.135]
	at com.communi.suggestu.scena.core.init.PlatformInitializationHandler.onInit(PlatformInitializationHandler.java:29) ~[scena-forge-1.0.95.jar%23441!/:1.0.95]
	at mod.chiselsandbits.forge.Forge.<init>(Forge.java:37) ~[chisels-and-bits-forge-1.3.135.jar%23300!/:1.3.135]
	at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[?:?]
	at java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499) ~[?:?]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:480) ~[?:?]
	at net.minecraftforge.fml.javafmlmod.FMLModContainer.constructMod(FMLModContainer.java:68) ~[javafmllanguage-1.19.2-43.2.14.jar%23411!/:?]
	at net.minecraftforge.fml.ModContainer.lambda$buildTransitionHandler$10(ModContainer.java:121) ~[fmlcore-1.19.2-43.2.14.jar%23410!/:?]
	at java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1804) [?:?]
	at java.util.concurrent.CompletableFuture$AsyncRun.exec(CompletableFuture.java:1796) [?:?]
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373) [?:?]
	at java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182) [?:?]
	at java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655) [?:?]
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622) [?:?]
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165) [?:?]
Caused by: java.lang.ClassNotFoundException: com.simibubi.create.content.contraptions.components.structureMovement.MovementBehaviour
	at jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641) ~[?:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?]
	at cpw.mods.cl.ModuleClassLoader.loadClass(ModuleClassLoader.java:137) ~[securejarhandler-2.1.4.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?]
	at cpw.mods.cl.ModuleClassLoader.loadClass(ModuleClassLoader.java:137) ~[securejarhandler-2.1.4.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?]
	... 24 more
[14:38:06] [modloading-worker-0/INFO]: Initialized Chisels&Bits-Forge client
```

### CODE REVIEW

File: settings.gradle [hunk 1]
```diff
@@ -18,7 +18,7 @@ pluginManagement.resolutionStrategy.eachPlugin {
     }
 }
 
-rootProject.name = "chisels-and-bits"
+rootProject.name = "Chisels-and-Bits"
```
Comment (marchermans) [inline settings.gradle]: Please revert this change.

**Review by marchermans — CHANGES_REQUESTED** [review]

**Comment by dima-dencep** [comment]
@marchermans

**Comment by marchermans** [comment]
Yes I am aware. I am busy in real and with porting.

**Comment by villainous-j** [comment]
Looking forward to having this merged! These mods are absolutely fantastic together.

If there's any way any of us can help make this happen please let us know!

**Pull request merged**